Tests/playwright.config.js.map

```diff
@@ -0,0 +1 @@
+{"version":3,"file":"playwright.config.js","sourceRoot":"","sources":["playwright.config.ts"],"names":[],"mappings":";;;;;;;;;;;;;AACA,yCAA2C;AAE3C;;;GAGG;AACH,8BAA8B;AAE9B;;GAEG;AACH,IAAM,MAAM,GAAyB;IACnC,OAAO,EAAE,aAAa;IACvB,0CAA0C;IAEzC,wCAAwC;IACxC,OAAO,EAAE,EAAE,GAAG,IAAI;IAClB,MAAM,EAAE;QACN;;;WAGG;QACH,OAAO,EAAE,KAAK;KACf;IACD,iFAAiF;IACjF,UAAU,EAAE,CAAC,CAAC,OAAO,CAAC,GAAG,CAAC,EAAE;IAC5B,sBAAsB;IACtB,OAAO,EAAE,OAAO,CAAC,GAAG,CAAC,EAAE,CAAC,CAAC,CAAC,CAAC,CAAC,CAAC,CAAC,CAAC;IAC/B,sCAAsC;IACtC,OAAO,EAAE,OAAO,CAAC,GAAG,CAAC,EAAE,CAAC,CAAC,CAAC,CAAC,CAAC,CAAC,CAAC,SAAS;IACvC,qEAAqE;IACrE,QAAQ,EAAE,MAAM;IAChB,wGAAwG;IACxG,GAAG,EAAE;QACH,oFAAoF;QACpF,aAAa,EAAE,CAAC;QAEhB,6DAA6D;QAC7D,oCAAoC;QAEpC,+FAA+F;QAC/F,KAAK,EAAE,gBAAgB;KACxB;IACD,2CAA2C;IAC3C,QAAQ,EAAE;QACR;YACE,IAAI,EAAE,UAAU;YAChB,GAAG,eACE,cAAO,CAAC,gBAAgB,CAAC,CAE7B;SACF;QAED,KAAK;QACL,sBAAsB;QACtB,YAAY;QACZ,sCAAsC;QACtC,QAAQ;QACR,MAAM;QAEN,IAAI;QACJ,oBAAoB;QACpB,WAAW;QACX,oCAAoC;QACpC,OAAO;QACP,KAAK;QAEL,oCAAoC;QACpC,IAAI;QACJ,2BAA2B;QAC3B,WAAW;QACX,6BAA6B;QAC7B,OAAO;QACP,KAAK;QACL,IAAI;QACJ,2BAA2B;QAC3B,WAAW;QACX,+BAA+B;QAC/B,OAAO;QACP,KAAK;QAEL,oCAAoC;QACpC,IAAI;QACJ,4BAA4B;QAC5B,WAAW;QACX,yBAAyB;QACzB,OAAO;QACP,KAAK;QACL,IAAI;QACJ,2BAA2B;QAC3B,WAAW;QACX,yBAAyB;QACzB,OAAO;QACP,KAAK;KACN;IAED,yEAAyE;IACzE,8BAA8B;IAE9B,yDAAyD;IACzD,eAAe;IACf,8BAA8B;IAC9B,gBAAgB;IAChB,KAAK;CACN,CAAC;AAEF,kBAAe,MAAM,CAAC"}
```

Comment:
Does this file have to be checked in? Isn't it generated in the build process?

Reply:
resolved